sites/next.skeleton.dev/src/pages/figma/index.astro

```diff
@@ -5,16 +5,157 @@ import LayoutRoot from '@layouts/LayoutRoot.astro';
+						<span class="text-surface-950-50">hard to align</span> with developers or clients?
+					</p>
+				</div>
+			</div>
+
+			<div class="basis-1/3 flex flex-row md:flex-col gap-4 p-6 bg-transparent shadow-lg border border-surface-200-800 rounded-container">
+				<div class="flex flex-col gap-12">
+					<SwatchBook size="32" className="opacity-80" />
+					<p class="text-surface-600-400 type-scale-5 md:type-scale-3">
+						Unsure how your design will look with<span class="text-surface-950-50"> a custom theme?</span>
+					</p>
+				</div>
+			</div>
+		</div>
+
+		<div class="pb-16">
+			<h2 class="h2 opacity-60">You’re not alone. We know these frustrations, and we’re here to help.</h2>
+		</div>
+
+		<h2 class="h2">We streamline your workflow by bringing design, development and your customers into perfect harmony.</h2>
+
+		<div class="flex flex-col gap-24 w-full py-12">
+			<div class="flex flex-col md:flex-row gap-6 w-full">
+				<div class="basis-1/2">
+					<Image src={ImgKit} class="rounded-container border border-surface-200-800" alt="components" />
+				</div>
+				<div class="flex flex-col gap-4 basis-1/2">
+					<h5 class="h5">Align quickly with prebuilt Skeleton components.</h5>
+					<p class="text-surface-800-200">
+						All components from the Skeleton design system are prebuilt and ready to use in Figma, saving you time and ensuring consistency.
+					</p>
+				</div>
+			</div>
+
+			<div class="flex flex-col-reverse md:flex-row gap-6 w-full">
+				<div class="flex flex-col gap-2 basis-1/2">
+					<h5 class="h5">Ensure consistency with theme imports and customizations.</h5>
+					<p class="text-surface-800-200">
+						Import your custom Skeleton themes directly into Figma using our plugin. Switch between dark and light modes or preview your
+						design with your theme instantly.
+					</p>
+				</div>
+
+				<div class="basis-1/2">
+					<Image src={ImgPlugin} class="rounded-container border border-surface-200-800" alt="components" />
+				</div>
+			</div>
+
+			<div class="flex flex-col md:flex-row gap-6 w-full">
+				<div class="basis-1/2">
+					<Image src={ImgTutorial} class="rounded-container border border-surface-200-800" alt="components" />
+				</div>
+				<div class="flex flex-col gap-2 basis-1/2">
+					<h5 class="h5">Iterate faster with tutorials designed for every step of the design process.</h5>
+					<p class="text-surface-800-200">Learn how to:</p>
+					<ul class="list-inside list-disc space-y-2 text-surface-800-200">
+						<li>Customize themes for your brand.</li>
+						<li>Create wireframes for quick iteration and client feedback.</li>
+						<li>Design developer-ready assets to streamline handoffs.</li>
+					</ul>
+				</div>
+			</div>
+		</div>
+
+		<div class="flex flex-col gap-6 py-12">
+			<blockquote class="blockquote text-3xl not-italic">
+				"...the Figma is very well put together and the most robust that I could find out of all the UI kits so our company chose to go with
+				Skeleton 🙂
```

Comment:
If you open a quote, you should probably close it, too ;) `"`

Reply:
Good catch - thx 👍